saltstack/base/salt/custom/usr/local/bin/custom-puppet.sh

```diff
@@ -14,7 +14,7 @@ yum install -y puppet-agent
 # Pull Puppet modules from Git
 # Note: It would be preferrable to use Puppetlabs r10k but baseline does not include required Ruby version
 mkdir /tmp/puppet-examples
-git clone https://github.com/gitowner/puppet-examples.git /tmp/puppet-examples
+git clone https://github.com/cloudera/puppet-examples.git /tmp/puppet-examples
```

Comment:
#CB-9301: The existing example link was pointed to gitowner's account in which Cloudera doesn't have any control. Pointed it cloudera.